hazelcast-client/src/test/java/com/hazelcast/client/io/IndependentBufferSizingTest.java

```diff
@@ -57,8 +57,8 @@ public void testClientBufferSize_SameAsForMemberMemberConnectionsByDefault() {
         HazelcastInstance client = HazelcastClient.newHazelcastClient();
 
         TcpIpConnection clientConnection = getClientConnection(server);
-        NonBlockingReadHandler readHandler = (NonBlockingReadHandler) clientConnection.getReadHandler();
-        NonBlockingWriteHandler writeHandler = (NonBlockingWriteHandler) clientConnection.getWriteHandler();
+        NonBlockingSocketReader readHandler = (NonBlockingSocketReader) clientConnection.getSocketReader();
```

Comment:
Shouldn't all those variables names be `socketReader` and `sockerWriter` after the change?